### PR TITLE
WIP upgrade to .NET Test XUnit preview3-build1039

### PR DESCRIPTION
**Pull request by devlead**
Just a WIP to test if they're any AppVeyor perf improvements with latest xunit tunner that uses an new batch API for reporting test results.

### CODE REVIEW

File: src/Cake.Common/Tools/Cake/CakeRunner.cs [hunk 1]
```diff
@@ -23,12 +23,34 @@ public sealed class CakeRunner : Tool<CakeSettings>
...
+                yield return directory.CombineWithFilePath("Cake.exe");
+                yield return directory.CombineWithFilePath("Cake.dll");
+            }
+        }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="CakeRunner"/> class.
         /// </summary>
-        /// <param name="fileSystem">The file system.</param>
+        /// <param name="fileSystem">The file system.</param>11
```
Comment (gep13) [inline src/Cake.Common/Tools/Cake/CakeRunner.cs]: @devlead what's this?

Comment (devlead) [inline src/Cake.Common/Tools/Cake/CakeRunner.cs]: that's me typing in the wrong window...

Comment (gep13) [inline src/Cake.Common/Tools/Cake/CakeRunner.cs]: @devlead ha ha, thought it might be something like that. Lucky it wasn't a password :stuck_out_tongue:

**Comment by devlead** [comment]
Issue lies with AppVeyor, will revisit once they've deployed.

**Pull request closed without merging**